packages/slate/src/interfaces/editor.ts

```diff
@@ -1107,8 +1126,18 @@ export const Editor = {
       return
     }
 
-    const [, from] = Editor.first(editor, at)
+    const pointBeforeLocation = Editor.before(editor, at, { voids })
+
+    const from = pointBeforeLocation && pointBeforeLocation.path
+
+    if (!from) {
+      return
+    }
+
     const [, to] = Editor.first(editor, [])
+
+    // The search location is from the start of the document to the path of
+    // the point before the location passed in
     const span: Span = [from, to]
```

Comment:
Will have to change this line if you accept my suggestion above.

```suggestion
    const span: Span = [pointBeforeLocation.path, to]
```